### PR TITLE
docs: Update links

### PR DESCRIPTION
**Pull request by aefhm**
## Description

Improve some results after running `linkinator`.

## TODO

- [x] Check preview

### CODE REVIEW

File: docs/general/oasis-network/why-oasis.md [hunk 1]
```diff
@@ -210,7 +209,6 @@ Over 200 articles have been written since the company came out of stealth mode.
 * [The New York Times: Building a World Where Data Privacy Exists Online](https://www.nytimes.com/2019/11/19/technology/artificial-intelligence-dawn-song.html)
 * [WIRED: Oasis Labs' Dawn Song on a Safer Way to Protect Your Data](https://www.wired.com/story/dawn-song-oasis-labs-data-privacy-wired25/)
 * [MIT Technology Review: How AI could save lives without spilling medical secrets](https://www.technologyreview.com/2019/05/14/239125/how-ai-could-save-lives-without-spilling-secrets/)
-* [Yahoo! Finance: Web3 Users will Control Their Data, Says Oasis Ecosystem Head](https://www.yahoo.com/now/web3-users-control-data-says-132837055.html)
```
Comment (aefhm) [inline docs/general/oasis-network/why-oasis.md:213]: I could not find one in on `archive.is` but do see Coin Rivet [here](https://coinrivet.com/web3-users-will-control-their-data-says-oasis-ecosystem-head/).

Comment (matevz) [reply docs/general/oasis-network/why-oasis.md:213]: Yeah, let's use the Coin Rivet's link.

**Review by matevz — APPROVED** [review]

File: docs/general/oasis-network/why-oasis.md [hunk 2]
```diff
@@ -161,16 +161,14 @@ If you want to build on Oasis, you can read more about the Ecosystem Fund [here]
...
-* [Anthem (by Chorus One)](https://anthem.chorus.one)
 * [Alethea.ai](https://medium.com/oasis-protocol-project/devaccelerator-spotlight-alethea-ai-aac8f854e436): A marketplace for AI-generated media
 * [Bankex](https://medium.com/oasis-protocol-project/devaccelerator-spotlight-bankex-9bb127f9e449): Send digital assets across your social networks
 * [Castalise](https://medium.com/oasis-protocol-project/devaccelerator-spotlight-castalise-347f155ab66f): Enabling privacy-preserving predictive analytics on pharmaceutical data
 * [Chainflow Mission Control](https://github.com/Chainflow/oasis-mission-control-grant)
 * [Dead Man’s Switch](https://medium.com/oasis-protocol-project/devaccelerator-spotlight-dead-mans-switch-32d07cdfc057): A decentralized, censorship-resistant tool for whistleblowers
 * [Luther.ai:](https://luther.ai)  Building artificial intelligence to Retain, Reinforce, Recall and ultimately augment your human memory.
 * [Oasis.Fish (by Stakefish)](https://oasis.fish/leaderboard/)
-* [Oasis Hubble (by Figment Networks)](https://www.figment.io/)
-* [Oasis Monitor (by Everstake)](https://oasismonitor.com)
```
Comment (aefhm) [inline docs/general/oasis-network/why-oasis.md:173]: Also noticed this one was deprecated while clicking around. cc @matevz

Comment (matevz) [reply docs/general/oasis-network/why-oasis.md:173]: Would be useful, if we searched through the docs for any Oasis Monitor mentions and remove the links.

Comment (aefhm) [reply docs/general/oasis-network/why-oasis.md:173]: Let's do that with https://github.com/oasisprotocol/docs/issues/987

**Pull request merged**